### PR TITLE
 installation & uninstallation instructions for Linux, MacOS and Android for V3

### PR DESCRIPTION
**Pull request by DaBigBlob**
## PREVIOUS PULL: https://github.com/pystardust/ani-cli/pull/753
# Install/Uninstall Instructions as per above pull

## Type of change

- [x] Bug fix
- [ ] Feature
- [x] Documentation update

## Description

installation instruction for Linux, MacOS and Android for V3.
&&
### POTENTIAL CRITICAL CHANGE
in the main `ani-cli` script
```diff
467c467
< project_root="$(dirname "$0")"
---
> project_root="$(dirname "$0" | sed 's/....$//')/share/ani-cli"
```
this is to comply with the unix file system hierarchy.
with the advent of ani-cli V3, it is no longer a single independent file.
therefore share/ani-cli/ directory is required.

## Checklist

- [x] any anime playing
- [x] bumped version
- [x] next, prev and replay work
- [x] quality works
- [x] downloads work
- [x] quality works with downloads
- [x] select episode -a and rapid resume work
- [x] syncplay -s works
- [x] autoplay, aka range selection, works

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,30 +57,26 @@ curl -s "https://raw.githubusercontent.com/pystardust/ani-cli/master/hist_transi
...
-
-Also consider ani-cli-git
-
-```sh
-yay -S ani-cli
-```
-
-### Other native packages
-
-[![Packaging status](https://repology.org/badge/vertical-allrepos/ani-cli.svg)](https://repology.org/project/ani-cli/versions)
```
Comment (port19x) [inline README.md]: These are important

File: README.md [hunk 2]
```diff
@@ -58,30 +57,26 @@ curl -s "https://raw.githubusercontent.com/pystardust/ani-cli/master/hist_transi
...
+How installation works:
+- removal of previous installation
+- downloading the essestial files to the appropriate share/ani-cli/ directory for your OS
+- creating a symbolic link from aforementioned share/ani-cli/ani-cli binary to the appropriate bin/ani-cli
+- watch anime 💪🏻
```
Comment (port19x) [inline README.md]: I'm unsure about the value of this section

File: README.md [hunk 3]
```diff
@@ -58,30 +57,26 @@ curl -s "https://raw.githubusercontent.com/pystardust/ani-cli/master/hist_transi
...
 Install dependencies [(See below)](#Dependencies)
 
 ```sh
-sudo curl -sL github.com/pystardust/ani-cli/raw/master/ani-cli -o /usr/local/bin/ani-cli &&
-sudo chmod +x /usr/local/bin/ani-cli
+sudo rm -rf "/usr/local/share/ani-cli" "/usr/local/bin/ani-cli" && \
+sudo git clone -b "master" "https://github.com/pystardust/ani-cli.git" "/usr/local/share/ani-cli" && \
+sudo ln -sf "/usr/local/share/ani-cli/ani-cli" "/usr/local/bin/ani-cli"
 ```
+*Note that while while running the above command, you'll be asked for your password.*
```
Comment (port19x) [inline README.md:84]: Not necessary. I assume people who find this project and run linux are aware of how sudo works

File: README.md [hunk 4]
```diff
@@ -58,30 +57,26 @@ curl -s "https://raw.githubusercontent.com/pystardust/ani-cli/master/hist_transi
...
+- watch anime 💪🏻
 
 ### Linux
 
 Install dependencies [(See below)](#Dependencies)
 
 ```sh
-sudo curl -sL github.com/pystardust/ani-cli/raw/master/ani-cli -o /usr/local/bin/ani-cli &&
-sudo chmod +x /usr/local/bin/ani-cli
+sudo rm -rf "/usr/local/share/ani-cli" "/usr/local/bin/ani-cli" && \
```
Comment (port19x) [inline README.md:80]: Is it a good idea to always do a "just in case" uninstall before an installation

**Review by port19x — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (DaBigBlob) [reply README.md:80]: on it

**Comment by justchokingaround** [comment]
looks ready for merge 👍

**Pull request merged**